mlflow/server/js/src/experiment-tracking/pages/experiment-scorers/SampleScorerOutputPanelContainer.tsx

```diff
@@ -164,12 +164,14 @@ const SampleScorerOutputPanelContainer: React.FC<SampleScorerOutputPanelContaine
   const hasInstructionsError = Boolean((errors as any).instructions?.message);
   const isRetrievalRelevance = llmTemplate === LLM_TEMPLATE.RETRIEVAL_RELEVANCE;
 
-  const isRunScorerDisabled = isCustomMode
```

Comment:
note: removed as there's no need to duplicate conditions both here and in `runScorerDisabledTooltip`